Core/Core/Views/GradeCircleView.swift

```diff
@@ -16,6 +16,29 @@
 
 import UIKit
 
+let topPadding: CGFloat = 12
+public class GradeCircleReusableView: UICollectionReusableView {
+//    @IBOutlet public weak var gradeCircleView: GradeCircleView?
+    public let gradeCircleView: GradeCircleView?
+
+    public override init(frame: CGRect) {
+        gradeCircleView = GradeCircleView(frame: CGRect(x: 0, y: topPadding, width: frame.width, height: frame.height))
+        super.init(frame: frame)
+        guard let gradeCircleView = gradeCircleView else {
+            return
+        }
+        addSubview(gradeCircleView)
+
+        let border = UIView(frame: CGRect(x: 16, y: frame.height, width: frame.width - 32, height: 1))
```

Comment:
do we want the height to be `1.0 / UIScreen.main.scale`?  Not a necessity.  Sometimes it makes borders just a hair smaller.

Reply:
Dave has had me change a few of those hairlines to be 1 pt. I think we mostly don't want hairlines in our current design.

Reply:
oh, good to know. Scratch that then.